orderer/common/cluster/block_fetcher.go

```diff
@@ -0,0 +1,104 @@
+package cluster
+
+import (
+	"github.com/hyperledger/fabric-protos-go/common"
+	"github.com/hyperledger/fabric/bccsp"
+)
+
+type suspectSet map[string]struct{}
+
+func (s *suspectSet) insert(entry string) {
+	if !s.has(entry) {
+		(*s)[entry] = struct{}{}
+	}
+}
+
+func (s *suspectSet) remove(entry string) {
```

Comment:
you don't need the "this" to be a pointer to the map. It can just be the type itself: 

func (s suspectSet) f() 

since the type is a map, you don't need a pointer to it.

Reply:
ok